requirements.txt

```diff
@@ -1,3 +1,4 @@
+pbr==5.5.1
```

Comment:
Why is this exact requirement needed? 
Is it exactly 5.5.1 or is >=5.5.1 also OK?

Reply:
Yes I think it should be okay, I was just testing and left it like that